cg/test/subjects/subjects.api.spec.js

```diff
@@ -91,9 +91,27 @@ describe('Tests of subject giving consent', () => {
     );
   });
 
+  it('Should return an error if the jwt token is expired', async done => {
+    const token = await subjectJWT.sign({ test: '10' }, { expiresIn: 1 });
```

Comment:
test: '10' means ?
expiresIn: 1 is one second or millisecond?

Reply:
test:10 means nothing, just a payload 
expiresIn1 is 1 second yes